### ISSUE TITLE
accuracy of pre-trained weights

### ISSUE THREAD

**Issue by serengil** [iwantooxxoox/keras-facenet#21]
Hello,

You mentioned that the pre-trained weights offer the accuracy score 0.938+-0.013. You converted the raw model from original OpenFace? Because none of the 4 shared models got this accuracy in the [official site](http://cmusatyalab.github.io/openface/models-and-accuracies/). If you trained the model from scratch you got this score on LFW data set?
